lib/ret/entity_state/create_message.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Ret.EntityState.CreateMessage do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Ret.Hub
+  alias Ret.EntityState.{CreateMessage, UpdateMessage}
+  @schema_prefix "ret0"
+  @primary_key {:entity_create_message_id, :id, autogenerate: true}
+
+  schema "entity_create_messages" do
+    field :nid, :string
+    field :create_message, :binary
```

Comment:
Yes. The binary field (here and in `UpdateMessage`) is the main data we are interested in storing to the database. It is the (client-side) `CreateMessage` that was sent to initialize an entity. In API responses, we send only this binary data to clients.

Documentation: https://github.com/mozilla/hubs/blob/6bbf167011917e4fc7d5821ce4e493940463e386/doc/org/build/networking.md#createmessage
Typescript type: https://github.com/mozilla/hubs/blob/7df9cf2e66fb6e73748e5351de4be1640fe8a796/src/utils/networking-types.ts#L12-L17  

We _could_ in theory store nothing but the `binary` blobs and the associated `hub_id`, but this would make it exceptionally difficult to find what we are looking for in any given query, since we would need to load an parse all the `binary`s to find the `networkId`. Instead, we save the `nid` and `hub_id` _next to_ the `binary`, so that we can index / search on them. 

It may be worth mentioning that custom clients will store custom data in these messages. We will not know the shape of the `initialData` within a (client-side) `CreateMessage`, or of the `data` within a (client-side) `UpdateMessage`.